.github/workflows/daily-test.yml

```diff
@@ -3,20 +3,28 @@ name: Daily Test
 on:
   schedule:
     # More information on cron https://crontab.guru/
-    # GitHub actions is using UTC time. Scheduling action at 5 am PST
-    - cron: '0 13 * * *'
```

Comment:
recommend not to run iOS and Android tests simultaneously, as our Saucelabs account has a limited capacity for concurrent sessions on physical devices.